pareto/other_models/CM_module/operational/screening_tool.ipynb

```diff
@@ -0,0 +1,801 @@
+      "\n",
+      "Solving bilinear problem...\n",
+      "\n",
+      " Successfully solved model\n",
+      "Arc cost:                               4269\n",
+      "Disposal cost:                       1103541\n",
+      "Freshwater cost:                     5302452\n",
+      "Treatment cost:                      1722170\n",
+      "Storage cost:                        1755032\n",
+      "Storage revenue:                     1737482\n",
+      "Lithium revenue:                      513365\n",
+      "Beneficial Reuse Revenue:             410692\n",
+      "Net cost:                            7225926\n"
+     ]
+    }
+   ],
+   "source": [
+    "model = build_qcp_br(data)\n",
+    "\n",
+    "# solving and displaying cost breakdown\n",
+    "model, _ = solving(model, tee=False)  # make tee=True if you want to see solver logs"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The model solved successfully which means that lithium recovery opportunities exist given the current infrastructure. Now we will try to assess whether more lithium revenue is possible."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assessing lithium revenue possibilities\n",
+    "\n",
+    "Now we will try to assess whether the current infrastructure is hindering lithium revenue opportunities. \n",
+    "\n",
+    "In the next block we will rerun the model, but this time to maximize lithium revenue without considering the impact that has on other operating costs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   ... running linear flow model\n",
+      "   ... running bilinear model\n",
+      "Max lithium revenue with existing infrastructure: 870608.1283902546\n",
+      "Arc cost:                              71795\n",
+      "Disposal cost:                       1117956\n",
+      "Freshwater cost:                     7751282\n",
+      "Treatment cost:                      2255071\n",
+      "Storage cost:                         314202\n",
+      "Storage revenue:                      214870\n",
+      "Lithium revenue:                      870608\n",
+      "Beneficial Reuse Revenue:             696487\n",
+      "Net cost:                            9728341\n"
+     ]
+    }
+   ],
+   "source": [
+    "max_inf_model = max_recovery_with_infrastructure(data)\n",
+    "print_results_summary(max_inf_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we can see the maximum Lithium revenue that can be attained given the existing infrastructure is $8,70,608/yr. \n",
+    "\n",
+    "Now we will compare this number to the maximum theoretical Li revenue that can be obtained from this network if we disregard the existing infrastructure."
```

Comment:
It's unclear to me what you mean by "if we disregard the existing infrastructure". Does this mean you're allowing the model to select a different desalination technology or treatment plant size at R01? What does the `treatment_unit` argument in the function `max_theoretical_recovery_flow_opt()` do? When you pass `"R01_IN"` for this argument, how exactly are you changing the model?

Later on, you say "As we can see the current infrastructure is not proving to be a major hindrance." It's unclear to me what you mean by this - it's a subjective statement, but the "max theoretical treatment revenue" is 35% higher than the "max lithium revenue with existing infrastructure" which seems like a significant difference to me.

I think all of this needs to be explained better.